### PR TITLE
remove the tricks for `paddle.fluid.layers.ops.func`

### PR DESCRIPTION
**Pull request by wadefelix**
### PR types
Others

### PR changes
Others

### Describe
1. remove the tricks for `paddle.fluid.layers.ops.func` https://github.com/PaddlePaddle/Paddle/pull/33634
2. refactor check_pr_approval, allow using github login-id

### CODE REVIEW

**Review by zhiqiu — DISMISSED** [review]
LGTM

File: tools/print_signatures.py [hunk 1]
```diff
@@ -30,7 +30,6 @@
 import pkgutil
 import logging
 import argparse
-import paddle
```
Comment (wadefelix) [inline tools/print_signatures.py:33]: 这个import删除并放到下面的函数中，是为了运行`test_print_signatures.py`测试用例，可以在不安装paddle的环境上执行此测试用例。

**Review by jzhang533 — APPROVED** [review]
LGTM
output of running `python3 tools/print_signatures.py paddle` may needs a clear format.

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**